infrastructure/cloud/environments/test/main.tf

```diff
@@ -255,30 +266,152 @@ module "ecs_api_td" {
-  subnet_ids     = module.subnets.app_subnets_ids
-  port           = module.ecs_api_td.port
+  source           = "../../modules/ECS/Service"
+  environment      = var.environment
+  app_name         = var.app_name
+  name             = "api"
+  ecs_cluster_id   = module.ecs_cluster.ecs_cluster.id
+  ecs_td_arn       = module.ecs_api_td.ecs_td_arn
+  tg_arn           = module.tg_api.tg_arn
+  sg_id            = data.aws_security_group.app_sg.id
+  subnet_ids       = module.subnets.app_subnets_ids
+  port             = module.ecs_api_td.port
+  ecs_cluster_name = module.ecs_cluster.ecs_cluster.name
+  max_capacity     = 1
+}
+
+# Create CloudWatch Alarms for Api ECS Service
+module "ecs_api_alarms" {
+  source       = "../../modules/Cloudwatch/Alarms"
+  environment  = var.environment
+  app_name     = var.app_name
+  namespace    = "AWS/ECS"
+  service_name = "api-ecs-service"
+  dimensions = {
+    ClusterName = module.ecs_cluster.ecs_cluster.name
+    ServiceName = module.ecs_api_service.service_name
+  }
+  alarm_configurations = [
+    {
+      name                = "cpu-high"
+      metric_name         = "CPUUtilization"
+      comparison_operator = "GreaterThanThreshold"
+      threshold           = var.alarm_cpu_threshold
+      evaluation_periods  = var.alarm_evaluation_periods
+      period              = var.alarm_period
+      statistic           = "Average"
+      description         = "CPU utilization sustained above ${var.alarm_cpu_threshold}% for ${var.alarm_evaluation_periods} minutes"
+      alarm_actions       = [module.sns_ecs_alerts.sns_topic_arn]
+    },
+    {
+      name                = "memory-high"
+      metric_name         = "MemoryUtilization"
+      comparison_operator = "GreaterThanThreshold"
+      threshold           = var.alarm_memory_threshold
+      evaluation_periods  = var.alarm_evaluation_periods
+      period              = var.alarm_period
+      statistic           = "Average"
+      description         = "Memory utilization above ${var.alarm_memory_threshold}% for ${var.alarm_evaluation_periods} minutes"
+      alarm_actions       = [module.sns_ecs_alerts.sns_topic_arn]
+    },
+    {
+      name                = "task-count-high"
+      metric_name         = "RunningTaskCount"
+      comparison_operator = "GreaterThanThreshold"
+      threshold           = 1
+      evaluation_periods  = 1
+      period              = var.alarm_period
+      statistic           = "Average"
+      description         = "API service has scaled up."
+      alarm_actions       = [module.sns_ecs_alerts.sns_topic_arn]
+    },
+    {
+      name                = "task-count-zero"
+      metric_name         = "RunningTaskCount"
+      comparison_operator = "LessThanThreshold"
+      threshold           = 1
+      evaluation_periods  = 1
+      period              = var.alarm_period
+      statistic           = "Average"
+      description         = "CRITICAL: API service has no running tasks - service is down"
```

Comment:
Properties are inconsistently parameterized; parameterized in some places and not in others.  Examples `threshold` and `evaluation_periods`.  There are a few areas like this.